lib/hermod/input_mutator.rb

```diff
@@ -6,8 +6,8 @@ def initialize(mutator_proc)
       @mutator_proc = mutator_proc
     end
 
-    def mutate!(values, attributes)
-      mutator_proc.call(values, attributes)
+    def mutate!(values, attributes, instance)
+      mutator_proc.call(values, attributes, instance)
```

Comment:
Given the tests already have a `proc` that doesn't care about the instance argument, I'd probably put a check against the `mutator_proc`'s arity here and only send the arguments it wants. (Which may also make this backwards compatible - winning?)

```ruby
lambda { |one, two| }.arity # => 2
lambda { |one, two, three| }.arity # => 3
```

```ruby
def mutate!(values, attributes, instance)
  if mutator_proc.arity == 2
    mutator_proc.call(values, attributes)
  else
    mutator_proc.call(values, attributes, instance)
  end
end
```